static/Functions.js

```diff
@@ -216,115 +214,124 @@ function processAttachments(isUpload, cb) {
 
 
       return httpRequest({
-        url: config.documentServiceUrl + "/api/Attachment",
+        // url: config.documentServiceUrl + "/api/Attachment",
+        url: "/api/attachment",
         method: 'POST',
-        contentType: "application/json; charset=utf-8",          
-        data: JSON.stringify(data),          
+        contentType: "application/json; charset=utf-8",
+        data: JSON.stringify(data),
         dataType: 'json',
-      }, function(err, response) {
-          if (err) return cb(err);
-        
-          // in this case the document service might return a result that contains an error, so also need to check this specifically
-          // TODO: revisit api on document service after rewriting in Node.js.
-          // if there's an error it should send back a statusCode != 200 to indicate that
-          if (response.isError) return cb(new Error('error uploading document: ' + response.message));
-
-          return cb(null, response);
+      }, function (err, response) {
+        if (err) return cb(err);
+
+        // in this case the document service might return a result that contains an error, so also need to check this specifically
+        // TODO: revisit api on document service after rewriting in Node.js.
+        // if there's an error it should send back a statusCode != 200 to indicate that
+        if (response.isError) return cb(new Error('error uploading document: ' + response.message));
```

Comment:
please remove this check, as it is not relevant now that we don't use the document service...
(you can read the comment that decorates it)

Reply:
removed